### ISSUE TITLE
`terraform import` doesn't work for `neon_role`

### ISSUE THREAD

**Issue by yurhasko** [kislerdm/terraform-provider-neon#108]
Hi there,

Thank you for opening an issue.

### Terraform Version
v1.6.3

### Terragrunt Version
v0.50.14

### Provider Version
0.6.1

### Affected Resource
- neon_role

### Terraform Configuration Files
```hcl
resource "neon_role" "my_neon_user" {
  project_id = neon_project.project.id
  name       = "my_neon_user"
  branch_id  = neon_project.project.default_branch_id
}
```

### Expected Behavior
Successful `terragrunt import neon_role.my_neon_user` with no changes. User exists in Neon and is actively used. Was imported following the `project/branch/username/` import schema (it was done the same way for the `neon_database`, and no issues occured there)

### Actual Behavior
After successful `terragrunt import neon_role.my_neon_user`, Terraform (Terragrunt) says that the resource should be recreated. Reason: "branch_id" and "project_id" force replacement.

### Important Factoids
Happens across all the environments that we used - we imported all resources into Terragrunt via `terragrunt import`, and it was successful and with "no changes", only `neon_user` is affected across all three environments. 

Occurs only with user that was created via Neon Console in the `main` (default) branch

**Comment by kislerdm** [comment]
Hello! Please share the terraform manifest and describe the state which you tried to import. Did I get it right that the project was created using a client other than terraform, i.e., web console, and an attempt to import the default role was made afterwards? Thanks!

**Comment by kislerdm** [comment]
@yurhasko hello! 

The issue was reproduced following these steps:
1. Create a project;
2. Create a role;
3. Remove role from the terraform state;
4. Import the role;
5. Run terraform plan.

**Want**: empty plan.
**Got**: non empty plan with the change of the role.
**Reason**: lack of logic to maintain the state of the project id and branch id which correspond to the role resource.

It will be fixed in v0.6.2. ETA: CW40.
